frontend/integration-tests/tests/deploy-image.scenario.ts

```diff
@@ -0,0 +1,43 @@
+/* eslint-disable no-undef, no-unused-vars */
+
+import { browser, $, element, by, ExpectedConditions as until, Key } from 'protractor';
+import * as _ from 'lodash';
+
+import { appHost, checkLogs, checkErrors, testName } from '../protractor.conf';
+import * as crudView from '../views/crud.view';
+
+describe('Deploy Image', () => {
+  afterEach(() => {
+    checkLogs();
+    checkErrors();
+  });
+
+  describe('Deployment Configs page', () => {
+    it('has Create button', async() => {
+      await browser.get(`${appHost}/k8s/ns/${testName}/deploymentconfigs`);
+      await crudView.isLoaded();
+      expect((element(by.buttonText('Create'))).isPresent()).toBe(true);
+    });
+
+    it('can be used to navigate to the Deploy Image page', async() => {
+      await $('#item-create').click().then(() => browser.actions().sendKeys(Key.ARROW_DOWN, Key.ENTER).perform());
+      await browser.wait(until.presenceOf($('.co-deploy-image')));
+      expect((element(by.cssContainingText('#resource-title', 'Deploy Image'))).isPresent()).toBe(true);
+    });
+  });
+
+  describe('Deploy Image page', () => {
+    it('can be used to search for an image', async() => {
+      await $('#imageName').sendKeys('mysql');
+      await $('.input-group-btn .btn-default').click();
+      await browser.wait(until.presenceOf($('.co-image-name-results__details')));
+      expect((element(by.cssContainingText('.co-image-name-results__heading', 'mysql'))).isPresent()).toBe(true);
+    });
+
+    it('can be used to create an app based on an image', async() => {
```

Comment:
We probably want to at least check that the created items we expect exist. You should be able to do something like

https://github.com/openshift/console/blob/master/frontend/integration-tests/tests/source-to-image.scenario.ts#L75-L83